packit_service/worker/checker/distgit.py

```diff
@@ -40,6 +56,15 @@ def pre_check(self) -> bool:
                 return False
 
             if self.data.event_dict["committer"] == "pagure":
+                if not self.pull_request:
+                    logger.debug(
+                        "Not able to get the pull request "
+                        "(may not be the head commit of the PR)."
+                    )
```

Comment:
Wouldn't it make more sense to return `True` here rather than in `contains_specfile_change()`?

Reply:
sure, I would swear it was there :D